### PR TITLE
Stringtable fixes

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
just some typo and punctuation fixes

### CODE REVIEW

**Review by killerswin2 — APPROVED** [review]

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -765,7 +765,7 @@
         <Original>Only Player Commander is allowed to move HQ assets.</Original>
       </Key>
       <Key ID="STR_A3A_fn_dialogs_movehqobj_no_far">
-        <Original>This asset cannot be moved more far away for its zone center.</Original>
+        <Original>This asset cannot be moved more far away from its zone center.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:768]: This asset cannot be moved further away from its zone center.

File: A3A/addons/core/Stringtable.xml [hunk 2]
```diff
@@ -515,7 +515,7 @@
         <Original>You are not the owner of this vehicle. Therefore, you cannot sell it.</Original>
       </Key>
       <Key ID="STR_A3A_fn_base_sellveh_no_suitable">
-        <Original>The vehicle you are looking is not suitable in our marketplace.</Original>
+        <Original>The vehicle you are looking at is not suitable in our marketplace.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:518]: The vehicle you are looking at is not suitable *for* our marketplace.

Arguable.

File: A3A/addons/core/Stringtable.xml [hunk 3]
```diff
@@ -946,19 +946,19 @@
...
-        <Original>Ilegible text encountered, need to decifer meaning from context.</Original>
+        <Original>Ilegible text encountered, need to decipher meaning from context.</Original>
       </Key>
       <Key ID="STR_A3A_fn_intel_encr_err_3_act">
-        <Original>Decifer codename</Original>
+        <Original>Decipher codename</Original>
       </Key>
       <Key ID="STR_A3A_fn_intel_encr_err_3_err">
-        <Original>Unkown codename encountered, need to decifer the codename reference.</Original>
+        <Original>Unkown codename encountered, need to decipher the codename reference.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:961]: Unknown not Unkown.

File: A3A/addons/core/Stringtable.xml [hunk 4]
```diff
@@ -1280,7 +1280,7 @@
         <Original>A group of smugglers have been arrested in %1 and they are about to be sent to prison. Go there and free them in order to make them join our cause. Do this before %2.</Original>
       </Key>
       <Key ID="STR_A3A_fn_mission_res_refu_text2">
-        <Original>A group of %3 supportes are hidden in %1 awaiting for evacuation. We have to find them before %2 does it. If not, there will be a certain death for them. Bring them back to HQ.</Original>
+        <Original>A group of %3 supporters are hidden in %1 awaiting for evacuation. We have to find them before %2 does it. If not, there will be a certain death for them. Bring them back to HQ.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:1283]: We have to find them before %2 does. (no "it")

File: A3A/addons/core/Stringtable.xml [hunk 5]
```diff
@@ -602,7 +602,7 @@
         <Original>Money Convoy</Original>
       </Key>
       <Key ID="STR_A3A_fn_convoy_craiact_prison_text">
-        <Original>A group os POW's is being transported from %1 to %3, and it's about to depart at %2. Try to intercept it. Kill or capture the truck driver to make them join you and bring them to HQ. Alive if possible.</Original>
+        <Original>A group of POW's is being transported from %1 to %3, and it's about to depart at %2. Try to intercept it. Kill or capture the truck driver to make them join you and bring them to HQ. Alive if possible.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:605]: POWs not POW's

**Pull request merged**